### DISCUSSION TITLE
Welcome to routerify Discussions!

### DISCUSSION THREAD

**Discussion by rousan** [routerify/routerify#44] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rousan** [comment]
Hi guys, We can discuss here for upcoming features and issues.

**Comment by rousan** [comment]
Released [v2.0.0-beta-2](https://github.com/routerify/routerify/releases/tag/v2.0.0-beta-2)!

**Reply by gsserge** [reply]
@rousan can you please make v2.0.0-beta-3 with the context request changes in? Thanks!

**Reply by rousan** [reply]
Sure

**Reply by gsserge** [reply]
Thanks! I'm going to run it for a couple of days and if all is well then I think we are in a good shape to make a proper 2.0 release.

**Reply by gsserge** [reply]
@rousan could you please yank 2.0.0-beta-3? `req.param()` is broken: for paths like `/api/todos/:id` with the request url `/api/todos/10` it will return `/api/todos/10` instead of just `10`. I'm investigating now, suspect https://github.com/routerify/routerify/pull/47. The fix is in https://github.com/routerify/routerify/pull/51

**Comment by gsserge** [comment]
I've created a Discord server for Routerify https://discord.gg/3zgb6YHZB2
@rousan @seanpianka @NikosEfthias @UndefinedBHVR @cpcloud you can join if you feel like it :)

**Reply by gsserge** [reply]
@rousan Hi! could you please make 2.0.1 release? We've got a nice optimization in the service code and an example fix in the readme. Thanks!

**Reply by rousan** [reply]
Sure, I will make a new release today.

**Reply by rousan** [reply]
Release should be done from which branch? `develop` or `master`?

**Reply by gsserge** [reply]
I've not merged the right stuff to `master` yet, so probably first `develop` to `master`, and then release from `master`.
Didn't think of that, will be better prepared next time.

**Reply by rousan** [reply]
So, `develop` branch will have all the development changes(unstable) and `master` branch will have latest stable code(latest release).

**Reply by gsserge** [reply]
Yes.

**Reply by rousan** [reply]
Giving compiling error:

```bash
error[E0658]: use of unstable library feature 'future_readiness_fns'
 --> src/service/router_service.rs:5:19
  |
5 | use std::future::{ready, Ready};
  |                   ^^^^^
  |
  = note: see issue #70921 <https://github.com/rust-lang/rust/issues/70921> for more information
```

**Reply by gsserge** [reply]
Rust 1.48+ is required.

**Reply by rousan** [reply]
Released v2.0.1.
